library/src/main/java/in/krharsh17/barview/BarView.java

```diff
@@ -130,6 +136,17 @@ private void populateBarView() {
         }
     }
 
+    public void setBackgroundColor(String color){
+        containerLayout.setBackgroundColor(Color.parseColor(color));
+    }
+
+    public void setBackgroundGradient(String startColor, String endColor, String direction){
```

Comment:
What's the use of `direction` here?

Reply:
it's for the direction of the gradient(left to right, top to bottom). currently, it is left to right. 
just want to ask, do we need to add options for direction for gradient or not?

Reply:
Yes, that would be preferable. A `gradient_direction` attribute for the barview, with two options: vertical & horizontal